database-seed/mongo-loader.js

```diff
@@ -19,14 +19,37 @@ const productImageScheme = mongoose.Schema({
 let Product = mongoose.model('ProductImage', productImageScheme);
 
 const seedDatabase = () => {
-    let documents = JSON.parse(retrieveData());
-    documents.forEach((document) => {
-        Product.updateOne({'productID':document.productID}, document, {upsert: true}, (err) => {
-            if (err) throw err;
+    dropDatabase()
```

Comment:
I like how you incorporated promises! The code looks a lot cleaner without the set time out